### ISSUE TITLE
Enforce consistent style for spacing in control structures

### ISSUE THREAD

**Issue by westonruter** [WordPress-Coding-Standards/WordPress-Coding-Standards#17]
The use of spaces around `if` and `foreach` and other constructs is not consistent. It would be good if we could force the same style with a sniff.

**Comment by JDGrimes** [comment]
> I think maybe we're still lacking here

#277 has fixed the inconsistency with colons, but the space between the closing parenthesis and the opening bracket doesn't seem to be enforced.
```php
if ( false ){ // Should be flagged, but isn't.

}
```

**Comment by JDGrimes** [comment]
Fixed by #366.
